README.md

```diff
@@ -90,6 +90,9 @@ A list of articles, tutorials, guides and videos about SwiftUI and Combine. **Fe
 ### Debug
 * [How to use Instruments to profile your SwiftUI code and identify slow layouts](https://www.hackingwithswift.com/quick-start/swiftui/how-to-use-instruments-to-profile-your-swiftui-code-and-identify-slow-layouts) by Paul Hudson
 
+### Opensource Components
```

Comment:
I don't think there is a need for a separate section for one project. Also, most of the projects mentioned in here are open source.